### DISCUSSION TITLE
Improve decision file's ''Decision Drivers'' & ''Pros/Cons'' sections

### DISCUSSION THREAD

**Discussion by Monstarrrr** [Monstarrrr/rebutify#87] [Ideas & Suggestions]
## Problem  

Currently, Decision Drivers and Pros/Cons are not being directly related when written.  
Also, Decision Drivers don't have a weight.  
This introduces too much bias (e.g. writting a bunch of Pros that are of low impact / low decision driver weight) and makes it harder to weight options.  

## Suggested solution  

Ensure Decision Drivers have a weight.
Ensure Decision Drivers and Pros/Cons are **directly** tied together using numbered list by adding the new structure below in the ``_template.md``:  

----  

### Decision drivers  

|          Criteria           |         Importance          |
|---------------------|-------------------------|
| [e.g. Reliability]  | [🔴 Critical \| 🟠 High \| 🟡 Medium] |
| [e.g. Weight]               | [🔴 Critical \| 🟠 High \| 🟡 Medium] |

#### Pros/Cons  
``<!-- Each criteria below must match the Decision Drivers criteria -->``  
| Criteria                    |                Pros              |              Cons                |
|---------------------|-------------------------|-------------------------|
| [e.g. Reliability]        | [e.g. Good because has 40k stars on github]<br/> [e.g. Good because weekly updates] | - |
| [e.g. Weight]            | - | [e.g. Bad because 32go] |

----

**Comment by seporterfield** [comment]
Making the pros and cons tie to the decision drivers is gonna be helpful! +1

**Comment by Monstarrrr** [comment]
Tables are annoying to edit, decided on a different way
